platforms/hyperledger-fabric/configuration/reset-network.yaml

```diff
@@ -39,7 +39,15 @@
         gitops: "{{ item.gitops }}"
         release_dir: "{{playbook_dir}}/../../../{{item.gitops.release_dir}}/{{ item.name|lower }}"
       loop: "{{ network['organizations'] }}"
-      when: network.type == 'fabric'
+
+    # Uninstalling Flux is needed so that everything is clean
+    # remove this if not needed
+    - name: Uninstall flux
```

Comment:
This should be in delete/flux_releases because then it works per org, and our assumption is that each org is a separate cluster i.e will have different flux deployed.

Reply:
The issue with keeping this in `delete/flux_releases` is that it deletes flux before some helmreleases if there are multiple orgs deployed to a single cluster.
I believe this prevents proper cleanup of the network.

Do you have an idea how to fix this properly?

Reply:
In that case, we need a separate role to delete just the flux and run that in a loop, so this code works, just it needs to run in network.orgs loop.